### PR TITLE
create seperate review moderation group and permissions (bug 1129395)

### PR DESCRIPTION
**Pull request by eviljeff**

### CODE REVIEW

**Comment by robhudson** [comment]
Is there a bug associated with this?

File: mkt/reviewers/views.py [hunk 1]
```diff
@@ -85,7 +85,7 @@
 log = commonware.log.getLogger('z.reviewers')
 
 
-def reviewer_required(region=None):
+def reviewer_required(region=None, moderator_too=False):
```
Comment (robhudson) [inline mkt/reviewers/views.py]: Heh. My vote would be just `moderator=False`.

File: mkt/reviewers/views.py [hunk 2]
```diff
@@ -112,7 +116,30 @@ def wrapper(request, *args, **kw):
...
+            if (acl.action_allowed(request, 'Apps', 'ModerateReview')):
+                return f(request, *args, **kw)
+            else:
+                raise PermissionDenied
+        return wrapper
+    # If decorator has no args, and is "paren-less", it's callable.
+    if callable(region):
+        return decorator(region)
+    else:
+        return decorator
```
Comment (robhudson) [inline mkt/reviewers/views.py]: I seem to remember a `permission_required` decorator that could be used instead of making another one?

**Comment by eviljeff** [comment]
There wasn't, so I logged a new bug.

File: mkt/reviewers/templates/reviewers/home.html [hunk 3]
```diff
@@ -9,6 +9,7 @@
     {{ reviewers_score_bar(types=point_types) }}
   </section>
 
+  {% if full_reviewer %}
```
Comment (robhudson) [inline mkt/reviewers/templates/reviewers/home.html]: Indent me...

File: mkt/reviewers/views.py [hunk 4]
```diff
@@ -94,13 +94,21 @@ def reviewer_required(region=None):
 
         Apps:Review
 
+    Moderator=True extends this to users in groups who have the permssion:
```
Comment (robhudson) [inline mkt/reviewers/views.py]: `moderator=True`?

**Comment by robhudson** [comment]
r+wc

**Pull request merged**